packages/wdio-types/src/Capabilities.ts

```diff
@@ -39,9 +39,16 @@ export interface ProxyObject {
     noProxy?: string[];
 }
 
+export interface WdioOptions {
+    /**
+     * Maximum number of total parallel running workers (per capability)
+     */
+    maxInstances?: number;
+}
```

Comment:
Instead of redefining, better do this:
```suggestion
interface WDIOCapabilityOptions extends Pick<TestrunnerOptions, 'specs' | 'exclude' | 'maxInstances'> {}
```

Reply:
done